modules/kind/00_mod.mk

```diff
@@ -15,7 +15,14 @@
 images_amd64 ?=
 images_arm64 ?=
 
-kind_version := v1.27.3
+kind_version := v1.29.0
 
-images_amd64 += docker.io/kindest/node:$(kind_version)@sha256:9dd3392d79af1b084671b05bcf65b21de476256ad1dcc853d9f3b10b4ac52dde
-images_arm64 += docker.io/kindest/node:$(kind_version)@sha256:de0b3dfe848ccf07e24f4278eaf93edb857b6231b39773f46b36a2b1a6543ae9
+# Goto https://github.com/kubernetes-sigs/kind/releases/tag/v0.20.0 and find the
```

Comment:
suggestion: This URL is correct now but won't be in the future. Future maintainers might copy+paste without noticing there's a version number in there.

```suggestion
# Goto https://github.com/kubernetes-sigs/kind/releases/tag/<KIND-VERSION> and find the
```